src/renderer/components/SettingsModal.tsx

```diff
@@ -348,6 +370,7 @@ export const SettingsModal = memo(({ isOpen, onClose }: SettingsModalProps) => {
                             <Tab>Environment</Tab>
                             <Tab>Python</Tab>
                             <Tab>Advanced</Tab>
+                            <Tab>DiscordRPC</Tab>
```

Comment:
I would probably put this setting in an existing tab rather than making it its own tab. After all, there's only ever going to be one setting for it. Maybe just put it in Advanced?